Makefile

```diff
@@ -124,23 +124,14 @@ helm:
 tidy:
 	@go mod tidy
 
-# ------------
-##@ Generators
-# ------------
-
-## Generate deepcopy code, kubernetes manifests and docs.
-generate:
-	./mage generate:all
-.PHONY: generate
-
 # ---------------------
 ##@ Testing and Linting
 # ---------------------
 
 ## Runs code-generators, checks for clean directory and lints the source code.
-lint:
+lint-mage:
```

Comment:
Let's keep it `lint` for now for the CI to pass.
While subscribing to boilerplate, let's override it.